graph/path/shortest.go

```diff
@@ -260,15 +274,21 @@ func newAllShortest(nodes []graph.Node, forward bool) AllShortest {
 	}
 }
 
+// at returns a slice of node indexed into p.nodes for nodes that are mid points
```

Comment:
This sentence is quite hard to parse so not sure but "a slice of nodes" (plural)?

Reply:
A common typo for me is -ed (_p.p._)-> -es (_pl._). I noticed what I though was this here and miscorrected the -es to -ed. I've fixed this and hopefully also clarified it (and below). It now means what I want it to mean, but still is difficult to parse.